integration_testing/features/super-admin/super-admin-bulk-import-users.feature

```diff
@@ -1,28 +1,35 @@
-Feature: Super Admin import users into a facility in a device with manage bulkexportusers command
-    Super Admin needs to be able to do a bulk import of the users and classes, with the information to enroll or assign coaches to the classes
+Feature: Super Admin bulk import users into a facility
+    Super Admin needs to be able to do a bulk import of users and classes using the command line
 
   Background:
     Given that the Kolibri server has one device with one or more facilities
 
-  Scenario: Execute the import command and review the report
-    When I run the 'kolibri manage bulkimportusers --dryrun --output-file=test.csv ' command in the Terminal
-    Then I see in the terminal a report containing the number of users to be created, deleted and updated. Same thing for the classes. I also see the possible errors the file contains
+  Scenario: Execute the bulkimportusers command and review the report
+    When I run the 'kolibri manage bulkimportusers --dryrun --output-file=test.csv' command in the Terminal
+    Then I see a report containing the number of users and classes to be created, deleted and updated
+      And I see the list of errors, if any
 
-  Scenario: Execute the import command and review the users and classes are created
+  Scenario: Execute the bulkimportusers command and review the users and classes are created
     When I run the 'kolibri manage bulkimportusers  --output-file=test.csv ' command in the Terminal
-    Then I see in the terminal a report containing the number of users to be created, deleted and updated. Same thing for the classes. I also see the possible errors the file contains.
-    Then I open the kolibri facility in the browser and check the users and classes have been created, with the coaches assigned and the learners enrolled.
-      And none of the previous users have been modified and continue assigned/enrolled as they were before.
+    Then I see a report containing the number of users and classes to be created, deleted and updated
+      And I see the list of errors, if any
+    When I open the facility in the browser
+      And I can see the new users and classes have been created, with the coaches assigned and the learners enrolled
```

Comment:
shouldn't it be `Then` instead of `And`?

Reply:
You got me! :bowing_woman: